config/settings.yml

```diff
@@ -41,10 +41,6 @@ sso:
   cookie_secure: false
   cookie_domain: localhost
 
-ssoe:
-  redirects:
-    myvahealth: https://ehrm-va-test.patientportal.us.healtheintent.com/
```

Comment:
Any corresponding devops PRs needed here, or credstash removals?

Reply:
thanks for reminding me.  the following block should be removed after this is deployed to production.  I'll update the linked issue with another task to do so

https://github.com/department-of-veterans-affairs/devops/blob/9e8b921829471b070c356d027dcd3dd8866f4af6/ansible/deployment/config/vets-api/prod-settings.local.yml.j2#L28-L30

Reply:
updated. https://github.com/department-of-veterans-affairs/va.gov-team/issues/13584